### PR TITLE
Many jobs at Monitor -- performance

### LINKED ISSUE #1501: Monitor performance issues with many jobs
The performance of the monitor seems to go down quite a bit when many jobs are registered. Especially just showing the scheduled page takes a long time, and adding an alert also seems to be very slow.

### PR DESCRIPTION
**Pull request by jhorcicka**
Fixes #1501 

For Scheduling page (listing of jobs) I used lighter version of "getSchedules" where I omitted properties loading. It does not seem to be necessary just for listing and then simple actions (run, delete, show history, ...). 

Time of listing 1000 copies of a simple job was decreased from 26s to 9s.

### CODE REVIEW

File: monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java [hunk 1]
```diff
@@ -322,7 +328,28 @@ public ScheduleDefinition eval(InputStream inputStream) {
 
         return schedule;
     }
+    
+    private ScheduleDefinition getScheduleWithoutProperties(final TenantIdentifier tenant, 
+            final JobIdentifier jobIdentifier) {
+        final JobContext jobContext = getJobContext(tenant, jobIdentifier);
+        final String groupName = jobContext.getGroupName();
+        final ScheduleDefinition schedule = new ScheduleDefinition(tenant, jobIdentifier, groupName);
```
Comment (jhorcicka) [inline monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java]: The old functionality is still there with the second argument set to "true". But I have to check hot folder.

Comment (jhorcicka) [inline monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java]: Hot folder seems to be working properly.

File: monitor/services/src/main/java/org/datacleaner/monitor/scheduling/SchedulingService.java [hunk 2]
```diff
@@ -42,7 +42,8 @@
 public interface SchedulingService extends RemoteService {
 
     @RolesAllowed({ SecurityRoles.VIEWER, SecurityRoles.SCHEDULE_EDITOR })
-    public List<ScheduleDefinition> getSchedules(TenantIdentifier tenant) throws DCSecurityException;
+    public List<ScheduleDefinition> getSchedules(TenantIdentifier tenant, boolean loadProperties) 
+            throws DCSecurityException;
```
Comment (ClaudiaPHI) [inline monitor/services/src/main/java/org/datacleaner/monitor/scheduling/SchedulingService.java]: Can we keep the old method as well and just redirected it to this one with "getSchedules(tenant, true)" ? Maybe someone else uses the method.

Comment (LosD) [inline monitor/services/src/main/java/org/datacleaner/monitor/scheduling/SchedulingService.java]: Not really a concern. It's a GWT class, so it not a public API (yet, hopefully at the end of the monitor rewrite, all interfaces to the web app are public)

**Comment by LosD** [comment]
LGTM!

**Pull request merged**